cordova-js-src/platform.js

```diff
@@ -58,6 +61,19 @@ module.exports = {
         bindButtonChannel('volumeup');
         bindButtonChannel('volumedown');
 
+        // The resume event is not "sticky", but it is possible that the event
+        // will contain the result of a plugin call. We need to ensure that the
+        // plugin result is delivered even after the event is fired (CB-10498)
+        var cordovaAddEventListener = document.addEventListener;
```

Comment:
This is really the original document.addListener. Maybe the name should be changed to make that more clear.

I think this is a simple and clean solution. However, I am a little weary about monkey patching `document.addEventListener`. I'm not sure what a better alternative would be though without adding unnecessary complexity.